tardis/io/tests/test_config_reader.py

```diff
@@ -190,35 +196,55 @@ def test_plasma_nlte_section_config(tardis_config_verysimple_nlte):
 
     Parameter
     ---------
-        `tardis_config_verysimple` : YAML File
+        `tardis_config_verysimple_nlte` : YAML File
+        `nlte_raw_model` : A simple model
+        `nlte_atom_data` : An example atomic dataset
 
     Result
     ------
         Assertion based on validation for specified values
     """
-    conf = Configuration.from_config_dict(
-        tardis_config_verysimple_nlte, validate=True, config_dirname="test"
-    )
     tardis_config_verysimple_nlte["plasma"]["continuum_interaction"][
         "species"
     ] = [
         "He I",
     ]
     tardis_config_verysimple_nlte["plasma"]["nlte_ionization_species"] = ["H I"]
-    with pytest.raises(ValueError) as ve:
-        nlte_ionization_species = tardis_config_verysimple_nlte["plasma"][
-            "nlte_ionization_species"
-        ]
-
-        for species in nlte_ionization_species:
-            if not (
-                species
-                in tardis_config_verysimple_nlte["plasma"][
-                    "continuum_interaction"
-                ]["species"]
-            ):
-                raise ValueError("Nlte ionization species not in continuum.")
-    assert ve.type is ValueError
+    with pytest.raises(PlasmaConfigError) as ve:
+        plasma = assemble_plasma(
+            tardis_config_verysimple_nlte, nlte_raw_model, nlte_atom_data
+        )
+    assert ve.type is PlasmaConfigError
+
+
+def test_plasma_nlte_exc_section_config(
+    tardis_config_verysimple_nlte, nlte_raw_model, nlte_atom_data
+):
+    """
+    Configuration Validation Test for Plasma Section of the Tardis Config YAML File.
+
+    Validates:
+        nlte_excitation_species: should be included in continuum_interaction
+
+    Parameter
+    ---------
+        `tardis_config_verysimple_nlte` : YAML File
+
+    Result
+    ------
+        Assertion based on validation for specified values
+    """
+    tardis_config_verysimple_nlte["plasma"]["continuum_interaction"][
+        "species"
+    ] = [
+        "He I",
+    ]
+    tardis_config_verysimple_nlte["plasma"]["nlte_excitation_species"] = ["H I"]
+    with pytest.raises(PlasmaConfigError) as ve:
+        plasma = assemble_plasma(
+            tardis_config_verysimple_nlte, nlte_raw_model, nlte_atom_data
+        )
+    assert ve.type is PlasmaConfigError
```

Comment:
```suggestion
```
The type of the Exception is already checked by the `pytest.raises`. I don't think checking the type after the with clause adds anything but I might be wrong.